docs/console/usage-billing.mdx

```diff
@@ -5,15 +5,6 @@ sidebar_label: Usage-based billing
 slug: usage-billing
 ---
 
```

Comment:
this section should stay and instead remove section `## Enable Usage-based Invoices`
 because this is no longer behind FT